src/Flickr.Net.Core/Flickrs/Flickr_Places_Async.cs

```diff
@@ -0,0 +1,616 @@
+
+        if (string.IsNullOrEmpty(placeId) && string.IsNullOrEmpty(woeId))
+        {
+            throw new FlickrException("Both placeId and woeId cannot be null or empty.");
+        }
+
+        if (!string.IsNullOrEmpty(woeId))
+        {
+            parameters.Add("woe_id", woeId);
+        }
+
+        if (!string.IsNullOrEmpty(placeId))
+        {
+            parameters.Add("place_id", placeId);
+        }
+
+        if (minTakenDate.HasValue && minTakenDate != DateTime.MinValue)
+        {
+            parameters.Add("min_taken_date", UtilityMethods.DateToMySql(minTakenDate.Value));
+        }
+
+        if (maxTakenDate.HasValue && maxTakenDate != DateTime.MinValue)
+        {
+            parameters.Add("max_taken_date", UtilityMethods.DateToMySql(maxTakenDate.Value));
+        }
+
+        if (minUploadDate.HasValue && minUploadDate != DateTime.MinValue)
+        {
+            parameters.Add("min_upload_date", UtilityMethods.DateToUnixTimestamp(minUploadDate.Value));
+        }
+
+        if (maxUploadDate.HasValue && maxUploadDate != DateTime.MinValue)
+        {
+            parameters.Add("max_upload_date", UtilityMethods.DateToUnixTimestamp(maxUploadDate.Value));
+        }
+
+        return await GetResponseAsync<TagCollection>(parameters, cancellationToken);
+    }
+}
+
+/// <summary>
+/// The flickr places.
+/// </summary>
+public interface IFlickrPlaces
+{
+    /// <summary>
+    /// Returns a list of places which contain the query string.
+    /// </summary>
+    /// <param name="query">The string to search for. Must not be null.</param>
+    /// <param name="cancellationToken"></param>
+    /// <return></return>
+    Task<PlaceCollection> FindAsync(string query, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Returns a place based on the input latitude and longitude.
+    /// </summary>
+    /// <param name="latitude">The latitude, between -180 and 180.</param>
+    /// <param name="longitude">The longitude, between -90 and 90.</param>
+    /// <param name="accuracy">The level the locality will be for.</param>
+    /// <param name="cancellationToken"></param>
+    /// <return></return>
+    Task<Place> FindByLatLonAsync(double latitude, double longitude, GeoAccuracy accuracy = GeoAccuracy.None, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Return a list of locations with public photos that are parented by a Where on Earth (WOE) or
+    /// Places ID.
+    /// </summary>
+    /// <param name="placeId">
+    /// A Flickr Places ID. (While optional, you must pass either a valid Places ID or a WOE ID.)
+    /// </param>
+    /// <param name="woeId">
+    /// A Where On Earth (WOE) ID. (While optional, you must pass either a valid Places ID or a WOE ID.)
+    /// </param>
+    /// <param name="cancellationToken"></param>
+    Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, string woeId = null, CancellationToken cancellationToken = default);
```

Comment:
Wouldn't it be better to overload this funtion (and functions similar to this) to make it clear that either "_placeId_", "_woeId_" (or both if possible) have to be set.
Adding functions like
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, CancellationToken cancellationToken = default)`
and
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string woeId, CancellationToken cancellationToken = default)`
for clarity and internally using a 
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, string woeId, CancellationToken cancellationToken = default)`
 function to still have the functionality in one place. The last function can also be public if wanted.
 
 This also requires some kind of special PlaceId or WoeId object to to get different signatures...